queries/c/textobjects.scm

```diff
@@ -53,3 +53,6 @@
 
 (preproc_else
   (_) @statement.outer)
+
+(parameter_list
+  (parameter_declaration) @parameter.inner)
```

Comment:
is this needed in this PR ?

Reply:
Usually you want to swap parameters.

Reply:
Should I add it for all languages?

Reply:
Is that possible for you ?

Reply:
Yes, tomorrow.

Reply:
Added the ones for python at https://github.com/nvim-treesitter/nvim-treesitter/pull/252

Reply:
As we saw in #252, many languages require a new predicate that allows excluding certain node types (here we need to exclude comments).

So adding @parameter should be separate PR.